blenderproc/python/utility/SetupUtility.py

```diff
@@ -81,9 +98,20 @@ def setup_utility_paths(temp_dir: str):
         Utility.temp_dir = resolve_path(temp_dir)
         os.makedirs(Utility.temp_dir, exist_ok=True)
 
+    @staticmethod
+    def clean_temp_dir() -> None:
```

Comment:
Sorry, there is still an issue with the `clean_temp_dir()`. So, in command_line.py, the original `clean_temp_dir()` needs to be used, as Utility.temp_dir does not exist outside the blender python env.
Afterwards, this `clean_temp_dir()` can again move back to Utility, as you had it before.

Reply:
Yes I also prefer option 1 :+1: Thanks!

Reply:
Done, now it seems from my testing that everything is working as it should.